docs/src/openapi-v1.yaml

```diff
@@ -1342,6 +1342,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ParasLeaseInfo'
+  /paras/head/included-candidates:
+    get:
+      tags:
+      - paras
+      summary: |
+        Get the heads of all the current included candidate parachains at the most recent finalized head.
```

Comment:
One little but important detail: I would write here 

"Get the heads of the included (backed and considered available) parachain candidates at the specified block height or at the most recent finalized head otherwise."

With this endpoint we are pulling the heads of the candidates that are included at a given relay chain block, but this does not imply we are retrieving all the included candidates at this height (state info).

I also added more details so that the reader understands the meaning of "included" here and fix it for the optional input `at`.

Reply:
This should be updated in the PR description too.

Reply:
Thanks for the input, I updated the description.